src/plugins/emailcomposer.ts

```diff
@@ -0,0 +1,63 @@
+import {Plugin, Cordova} from './plugin';
+
+/**
+ * Email object for Opening Email Composer
+ */
+export interface email {
+  to: string | Array<string>,
+  cc: string | Array<string>,
+  bcc: string | Array<string>,
+  attachments: Array<any>,
+  subject: string,
+  body: string,
+  isHtml: boolean
+}
+
+/**
+ * @name Email Composer
+ * @description
+ *
+ * Requires Cordova plugin: cordova-plugin-email-composer. For more info, please see the [Email Composer plugin docs](https://github.com/katzer/cordova-plugin-email-composer).
+ *
+ * @usage
+ * ```ts
+ * import {EmailComposer} from 'ionic-native';
+ *
+ *
+ * let email = {
+ *   to: 'max@mustermann.de',
+ *   cc: 'erika@mustermann.de',
+ *   bcc: ['john@doe.com', 'jane@doe.com'],
+ *   attachments: [
+ *     'file://img/logo.png',
+ *     'res://icon.png',
+ *     'base64:icon.png//iVBORw0KGgoAAAANSUhEUg...',
+ *     'file://README.pdf'
+ *   ],
+ *   subject: 'Cordova Icons',
+ *   body: 'How are you? Nice greetings from Leipzig',
+ *   isHtml: true
+ * };
+ *
+ * // Send a text message using default options
+ * EmailComposer.send(email);
+ *
+ * ```
+ */
+@Plugin({
+  plugin: 'cordova-plugin-email-composer',
+  pluginRef: 'emailComposer',
+  repo: 'https://github.com/katzer/cordova-plugin-email-composer.git',
+  platforms: ['Android', 'iOS', 'Windows Phone 8']
+})
+export class EmailComposer {
+
```

Comment:
Missed that when I looked at the ngCordova docs for the plugin will work on adding this. 
Thoughts on whether as first pass we need to implement the overload with the urischeme passed in as well?